setup.py

```diff
@@ -91,7 +91,6 @@ def get_package_data(package):
     packages=get_packages(package),
     package_data=get_package_data(package),
     cmdclass={'test': PyTest},
-    install_requires=install_requires,
```

Comment:
The `install_requires` were removed in @rlucioni 's PR to make the dependencies optional based on which auth method was being used:
https://github.com/jpadilla/django-rest-framework-oauth/pull/12